### ISSUE TITLE
Swift Compile Errors

### ISSUE THREAD

**Issue by roberttolton** [radarlabs/capacitor-radar#13]
When using this SDK in my Capacitor app, I'm met with the following errors when trying to build in XCode:

```
node_modules/capacitor-radar/ios/Plugin/Plugin.swift:429:47: Use of undeclared type 'RadarRegion'
node_modules/capacitor-radar/ios/Plugin/Plugin.swift:237:15: Type 'RadarEventType' has no member 'userEnteredRegionCountry'
node_modules/capacitor-radar/ios/Plugin/Plugin.swift:239:15: Type 'RadarEventType' has no member 'userExitedRegionCountry'
node_modules/capacitor-radar/ios/Plugin/Plugin.swift:241:15: Type 'RadarEventType' has no member 'userEnteredRegionState'
node_modules/capacitor-radar/ios/Plugin/Plugin.swift:243:15: Type 'RadarEventType' has no member 'userExitedRegionState'
node_modules/capacitor-radar/ios/Plugin/Plugin.swift:245:15: Type 'RadarEventType' has no member 'userEnteredRegionDMA'
node_modules/capacitor-radar/ios/Plugin/Plugin.swift:247:15: Type 'RadarEventType' has no member 'userExitedRegionDMA'
node_modules/capacitor-radar/ios/Plugin/Plugin.swift:333:32: Value of type 'RadarUser' has no member 'country'
node_modules/capacitor-radar/ios/Plugin/Plugin.swift:336:30: Value of type 'RadarUser' has no member 'state'
node_modules/capacitor-radar/ios/Plugin/Plugin.swift:339:28: Value of type 'RadarUser' has no member 'dma'
node_modules/capacitor-radar/ios/Plugin/Plugin.swift:342:35: Value of type 'RadarUser' has no member 'postalCode'
node_modules/capacitor-radar/ios/Plugin/Plugin.swift:464:32: Value of type 'RadarEvent' has no member 'region'
```

XCode Version: `11.2.1`

**Comment by nickpatrick** [comment]
Which Radar SDK version are you pulling? These were added in 2.1.4 https://github.com/radarlabs/radar-sdk-ios/releases/tag/2.1.4 and 2.1.5 https://github.com/radarlabs/radar-sdk-ios/releases/tag/2.1.5.

**Comment by roberttolton** [comment]
Following all the installation instructions on a new project, I'm using:

`capacitor-radar 0.0.10`
`RadarSDK 2.1.3`

`2.1.3` of course being lower than the versions you mentioned, I tried changing the line:

`pod 'RadarSDK', '~> 2.1.0'` to `pod 'RadarSDK', '~> 2.1.5'`

Which gave me:

```
[!] CocoaPods could not find compatible versions for pod "RadarSDK":
  In Podfile:
    CapacitorRadar (from `../../node_modules/capacitor-radar`) was resolved to 0.0.10, which depends on
      RadarSDK

    RadarSDK (~> 2.1.5)

None of your spec sources contain a spec satisfying the dependencies: `RadarSDK (~> 2.1.5), RadarSDK`.

You have either:
 * out-of-date source repos which you can update with `pod repo update` or with `pod install --repo-update`.
 * mistyped the name or version.
 * not added the source repo that hosts the Podspec to your Podfile.

Note: as of CocoaPods 1.0, `pod repo update` does not happen on `pod install` by default.
```

So I updated cocoapods, and then re-run `pod-install`:

```

**Comment by nickpatrick** [comment]
Great. Yep, sometimes you need to update your repos. We'll pin to the latest version in a future release.
